tests/test_mcmc.py

```diff
@@ -63,26 +63,31 @@ def fix_state(request):
 
 
 @pytest.fixture(
-    params=[(0, 4000), (2000, 4000), (0, 6000), (2000, 6000)],
-    ids=["n_burn=0,n_iter=4000", "n_burn=non-zero,n_iter=4000", "n_burn=0,n_iter=6000", "n_burn=non-zero,n_iter=6000"],
-    name="nburn_niter",
+    params=[(0, 4000, 1), (2000, 4000, 5), (0, 6000, 10), (2000, 6000, 1)],
+    ids=[
+        "n_burn=0,n_iter=4000, n_thin=1",
+        "n_burn=non-zero,n_iter=4000,n_thin=5",
+        "n_burn=0,n_iter=6000, n_thin=10",
+        "n_burn=non-zero,n_iter=6000,n_thin=1",
+    ],
+    name="mcmc_settings",
 )
-def fix_nburn_niter(request):
+def fix_mcmc_settings(request):
     """Define the initial state for the MCMC."""
-    [n_burn, n_iter] = request.param
-    nburn_niter = {"nburn": n_burn, "niter": n_iter}
+    [n_burn, n_iter, n_thin] = request.param
+    fix_mcmc_settings = {"nburn": n_burn, "niter": n_iter, "nthin": n_thin}
```

Comment:
I'd maybe just call this variable `mcmc_settings` rather than `fix_mcmc_settings`.